### PR TITLE
Python 2.0

### PR DESCRIPTION
**Pull request by ErisMik**

### CODE REVIEW

File: demo/python/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 numpy
 pvporcupine==1.9.5
```
Comment (mrrostam) [inline demo/python/requirements.txt:2]: pvporcupine==2.0.0

i know it's not published yet, but the demo is not gonna also work with the old one as well

File: demo/respeaker/setup.py [hunk 2]
```diff
@@ -33,7 +33,7 @@
     long_description_content_type="text/markdown",
     url="https://github.com/Picovoice/porcupine",
     packages=["ppnrespeakerdemo"],
-    install_requires=["pvporcupine==1.9.5", "spidev", "gpiozero"],
+    install_requires=["pvporcupine==1.9.5", "spidev", "gpiozero", "pvrecorder==1.0.2"],
```
Comment (mrrostam) [inline demo/respeaker/setup.py:36]: pvporcupine==2.0.0

Comment (mrrostam) [reply demo/respeaker/setup.py:36]: also `version="2.0.0"`

File: demo/tkinter/requirements.txt [hunk 3]
```diff
@@ -1,2 +1,2 @@
-pyaudio
 pvporcupine==1.9.5
```
Comment (mrrostam) [inline demo/tkinter/requirements.txt:1]: pvporcupine==2.0.0

File: demo/python/requirements.txt (hunk 1 above)
Comment (mrrostam) [reply demo/python/requirements.txt:2]: also increase the version in `setup.py` for the binding

File: demo/python/porcupine_demo_mic.py [hunk 4]
```diff
@@ -52,6 +53,7 @@ def __init__(
 
         super(PorcupineDemo, self).__init__()
 
+        self.access_key = access_key
```
Comment (ksyeo1010) [inline demo/python/porcupine_demo_mic.py:56]: self._access_key to keep it consistent with others?

**Review by mrrostam — APPROVED** [review]

**Review by ksyeo1010 — APPROVED** [review]

**Pull request merged**